src/renderer/components/+workloads-statefulsets/statefulset-menu.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import React from "react";
+import type { KubeObjectMenuProps } from "../kube-object-menu";
+import type { StatefulSet, StatefulSetApi } from "../../../common/k8s-api/endpoints";
+import { MenuItem } from "../menu";
+import { Icon } from "../icon";
+import { Notifications } from "../notifications";
+import { withInjectables } from "@ogre-tools/injectable-react";
+import statefulSetApiInjectable from "../../../common/k8s-api/endpoints/stateful-set.api.injectable";
+import type { OpenConfirmDialog } from "../confirm-dialog/open.injectable";
+import openConfirmDialogInjectable from "../confirm-dialog/open.injectable";
+
+export interface StatefulSetMenuProps extends KubeObjectMenuProps<StatefulSet> {}
+
+interface Dependencies {
+  statefulsetApi: StatefulSetApi;
+  openConfirmDialog: OpenConfirmDialog;
+}
+
+const NonInjectedStatefulSetMenu = ({
+  statefulsetApi,
+  object,
+  toolbar,
+  openConfirmDialog,
+}: Dependencies & StatefulSetMenuProps) => (
+  <>
+    <MenuItem
+      onClick={() => openConfirmDialog({
+        ok: async () =>
+        {
+          try {
+            await statefulsetApi.restart({
+              namespace: object.getNs(),
+              name: object.getName(),
+            });
+          } catch (err) {
+            Notifications.checkedError(err, "Unknown error occured while restarting statefulset");
```

Comment:
Please use the `showCheckedErrorNotificationInjectable` here.

Reply:
Sorry I tried to look around for examples but I'm not able to figure out how to use that function instead of `Notifications.checkedError`. The files that use it seem to use completely differently constructors. Apologies, if you are able to show me how to do that I'll raise some PRs afterwards to switch over the other menus to use the same function as well.

Reply:
All good. Thanks for looking around. It is pretty easy, just make the following changes:

1. Add a new line after line 20 above with the contents: `showCheckedErrorNotification: ShowCheckedErrorNotification;`
2. Add a new line after line 27 above with the contents: `showCheckedErrorNotification,`
3. Change line 40 to read `showCheckedErrorNotification(err, ...);`
4. Add a new line after line 67 below with the contents: `showCheckedErrorNotification: di.inject(showCheckedErrorNotificationInjectable),`
5. Use the IDE to auto import the correct things.

Reply:
Thanks a lot, I've made the changes above.